### PR TITLE
Fix issues #5 and #13

### LINKED ISSUE #5: Anchors should be anchors, links should be links, and includes should be includes.
_Everyday terminology_ refers to the semantic `<a>` element as a "link" _OR_ an "anchor", giving the tag a double meaning.

Specs refer to a "link" as what _everyday terminology_ refers to as an "include".

This is a serious barrier of entry for anyone trying to learn HTML. "I've been told links are what take you to new documents my whole life, but why are they something completely different in HTML land?", students ask. "Because the HTML gods made it so", I tell them.

Everyday peoples understand the (HUGE) difference between a website "link" and an "anchor". It's an easy concept, conceptually. However, once they get into HTML, real-world understanding doesn't map to the reality ( _cough_ poorly invented conventions _cough_ ) of HTML-fairytale-land.

I've always had this question, but figured there was some mystical force at work that justified such idiotic terminology. After exploring the deepest caverns of HTML I can finally put that myth to rest, and take a stand against this retardation!

A ludicrous proposal: a "link" actually be what normal people understand a "link" to be. This means `<html:a>` or possibly `<html:anchor>` will only be used for ANCHORS and `<html:link>` will only be used for LINKS in the true sense of the term. Now that `<html:link>` is taken, we must endow a new term for the _inclusion_ of source files; I propose `<html:include>`.

Please poke holes in this philosophy if you can, because from where I sit it seems sound; but perhaps I missed one of those secret underground passageways that leads you to the bonus level where there's flying pigs and this all makes sense.

PS: "HTML6" seems like a splendid idea, regardless of if my points are valid.

### LINKED ISSUE #13: Cleaner meta tags
Allow more than one meta name to share the same value.

``` HTML
<html:meta types="description, twitter:description, og:description" value="This is an example of description for Google, Twitter, and Facebook">
```

### PR DESCRIPTION
**Pull request by cshaa**
Update html:meta, split html:link and html:a to html:alter, html:include, html:link and html:anch

Closes #13
Closes #5

### CODE REVIEW

**Comment by OscarGodson** [comment]
@m93a without a target tho how would JavaScript know the target of the element? A common use case: http://www.ajaxf1.com/tutorial/ajax-file-upload-tutorial.html

File: README.md [hunk 1]
```diff
@@ -148,23 +153,50 @@ _Example:_
 </html:html>
 ```
 
-##### `<html:link>`
+##### `<html:alter>`
 
-This links external documents and scripts such as CSS, JavaScript, RSS, favicons, etc. to the current document. Equivalent to the current `<link>` tag. This tag takes the following attributes:
+This links document's alternatives in other formats, eg. PDF, RTF, RSS or Atom feed. It's similar to the current HTML `link` tag.
```
Comment (OscarGodson) [inline README.md:158]: Why would you have a different link just for documents? Why shouldn't the browser handle the displaying of these like it does now? What's considered an "alternative document" and how would you know what is and isn't one?

File: README.md [hunk 2]
```diff
@@ -148,23 +153,50 @@ _Example:_
...
+<!DOCTYPE html>
+<html:html>
+  <html:head>
+    <html:title>HTML6 Spec Version 0.1</html:title>
+    <html:alter src="rss?feed=1" title="RSS Feed" type="application/rss+xml">
+  </html:head>
+</html:html>
+```
+
+##### `<html:include>`
```
Comment (OscarGodson) [inline README.md:180]: +1

File: README.md (hunk 1 above)
Comment (cshaa) [inline README.md:158]: Current "link" was made to do that but little by little it became more of "include" than a real link. If you use <link> as a reference to RSS, the browser just shows _feed_ button and doesn't even download the file. But if you use it as a link to css stylesheet, browser downloads the file, includes it to the page and changes its content.

In layman's terms: `include` changes the document, `alter` doesn't

---

Or maybe we can use `meta` instead of `alter`?

Comment (OscarGodson) [inline README.md:158]: So, basically this link just forces a download?

**Comment by cshaa** [comment]
Now, my pull request should point to `_includes/HTML6.md`, not `README.md`

File: _includes/HTML6.md [hunk 3]
```diff
@@ -187,15 +219,40 @@ _Example:_
...
 
-Attributes available to the `<a>` tag are:
+_Normative:_  
+Attributes available to this tag are:
 
-- `href`
-- `name`
-- `target` (can be `blank`, `parent`, `top` or `self`)
+- `href`: The link to the target document, can be relative or absolute.
+- `target`: Which window will open the document (can be `blank`, `parent`, `top` or `self`, default is `self`). **This attribute is obsolete**.
```
Comment (cshaa) [inline _includes/HTML6.md:231]: OscarGodson:

```
@m93a without a target tho how would JavaScript know the target of the element?
A common use case:
http://www.ajaxf1.com/tutorial/ajax-file-upload-tutorial.html
```

Comment (cshaa) [inline _includes/HTML6.md:231]: @OscarGodson 
I would make a `<button>` with onclick and use JavaScript XMLHttpRequest and File API.
A common use case:
http://www.matlus.com/html5-file-upload-with-progress/

Comment (OscarGodson) [inline _includes/HTML6.md:231]: If you only have to support new browsers, that'd be fine :)

File: README.md (hunk 1 above)
Comment (OscarGodson) [inline README.md:158]: I'm assuming, yes? If so there's no need for an entirely different element for that. Just have a like `download` attribute that's already in HTML now.

File: _includes/HTML6.md (hunk 3 above)
Comment (cshaa) [inline _includes/HTML6.md:231]: Old browsers won't support HTML6 :)

Comment (philandy) [inline _includes/HTML6.md:231]: Oscar HTML6 doesn't have to worry about backwards compatibility because that would be up to the browsers.

**Comment by cshaa** [comment]
@OscarGodson would you merge it to the master, please?

**Comment by OscarGodson** [comment]
I'm still waiting on responses from you in the PR :) I'm not sold on your explanation of link and removing `a`.

**Comment by cshaa** [comment]
Oh, that's a reaction to #5. It's about splitting `html:a` to `html:anch` and `html:link`.

File: README.md (hunk 1 above)
Comment (cshaa) [inline README.md:158]: include = make it a part of this document

File: _includes/HTML6.md [hunk 4]
```diff
@@ -187,15 +219,40 @@ _Example:_
...
+  <html:head>
+    <html:title>HTML6 Spec Version 0.1</html:title>
+  </html:head>
+  <html:body>
+    <html:link href="http://google.com">Go to google.com!</html:link>
+  </html:body>
+</html:html>
+```
+
+##### `<html:anch>`
```
Comment (OscarGodson) [inline _includes/HTML6.md]: You could make this into it's own PR and ask people how they like it, but I don't see any benefit to this over just allowing `a`/`link`. In both cases it makes semantic sense. If we had a `link` element it'd mean `link to this page, and this section`. You'd also need links like `http://someothersite.com/#foo` to work anyway so making them totally different links seems confusing.

File: _includes/HTML6.md [hunk 5]
```diff
@@ -148,23 +153,50 @@ _Example:_
 </html:html>
 ```
 
-##### `<html:link>`
+##### `<html:alter>`
```
Comment (OscarGodson) [inline _includes/HTML6.md:156]: I see. Makes sense to me now. I'll +1

File: _includes/HTML6.md (hunk 4 above)
Comment (OscarGodson) [inline _includes/HTML6.md]: I think if you remove this and make it it's own PR and get feedback I'll accept the PR :)

Comment (cshaa) [inline _includes/HTML6.md]: Ok, rollback to html:a done. Adding "for" attribute to it so you can link elements using css selectors. It would be great to have a possibility to link somebody to the third `div.rc` of `google.com/#q=foo`.

**Comment by cshaa** [comment]
@OscarGodson is there anything you don't agree with? :)  
Actually, writing specifications began to be a good fun for me, so I'm ready to correct it.

**Comment by risenomore** [comment]
"This begins an HTML's head. Equivelent to the current &lt;html&gt; tag."

Should read: "This begins an HTML's head. Equivelent to the current <b>&lt;head&gt;</b> tag."
